### PR TITLE
Enhance Wiz for CDR support

### PR DESCRIPTION
**Pull request by CohenIdo**
solves:
- https://github.com/elastic/security-team/issues/9829

## Summary
* Update vulnerability data stream's ingest pipeline to support CDR.
* Update vulnerability data stream's mappings according to the new fields.
* Create latest transform for the Wiz vulnerabilities.

### CODE REVIEW

File: packages/wiz/manifest.yml [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
+version: "1.6.0"
 description: Collect logs from Wiz with Elastic Agent.
 type: integration
 categories:
   - security
   - cloudsecurity_cdr
 conditions:
   kibana:
-    version: "^8.13.0"
+    version: "^8.16.0"
```
Comment (CohenIdo) [inline packages/wiz/manifest.yml:12]: The transform requires Elasticsearch privilege which is granted in [this PR](https://github.com/elastic/elasticsearch/pull/112192) and will we available from version 8.16.

File: packages/wiz/changelog.yml [hunk 2]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.5.1"
```
Comment (maxcold) [inline packages/wiz/changelog.yml:2]: I think the change is quite big for the patch release, as it is not breaking, minor bump should do

File: packages/wiz/elasticsearch/transform/latest_cdr_vulnerabilities/fields/base-fields.yml [hunk 3]
```diff
@@ -0,0 +1,20 @@
+- name: data_stream.type
+  type: constant_keyword
+  description: Data stream type.
+- name: data_stream.dataset
+  type: constant_keyword
+  description: Data stream dataset.
+- name: data_stream.namespace
+  type: constant_keyword
```
Comment (maxcold) [inline packages/wiz/elasticsearch/transform/latest_cdr_vulnerabilities/fields/base-fields.yml:8]: As [per RFC](https://docs.google.com/document/d/1bEH3AMF_pwk-AYfq-X-2po39ydHT4yEpLT2mamhyP3Y/edit) we want to make `data_stream.namespace` type `keyword`. This should allow us to support namespaces in the future and not run into this issue https://github.com/elastic/kibana/issues/162889#issuecomment-1978012763

File: packages/wiz/manifest.yml (hunk 1 above)
Comment (maxcold) [reply packages/wiz/manifest.yml:12]: what about adding transform as a separate PR to enable CDR fields on the original data stream without the need for the newest version of the stack? I plan to release the Cloud Configurations Finding data stream without the transform initially and then add the transform as it requires the stack version bump. We could do the same for vulnerabilities. Also will be easier test and rollback if needed, while having new fields on the data stream without risks (even though we won't be able to use them directly, they might become handy for features like timeline and alert creation)

File: packages/wiz/data_stream/vulnerability/elasticsearch/ingest_pipeline/default.yml [hunk 4]
```diff
@@ -27,20 +27,20 @@ processors:
       target_field: event.original
```
Comment (maxcold) [inline packages/wiz/data_stream/vulnerability/elasticsearch/ingest_pipeline/default.yml:27]: we also need to add 
```
  - set:
      field: observer.vendor
      tag: set_observer_vendor
      value: 'Wiz'
```

to solve for https://github.com/elastic/kibana/issues/188802

File: packages/wiz/manifest.yml (hunk 1 above)
Comment (CohenIdo) [reply packages/wiz/manifest.yml:12]: Good idea, I will split this PR for two others: 
[Here](https://github.com/elastic/integrations/pull/10892) is the first one for handling data stream changes (I also resolved your comments there), and I'll create another one for the transform.

**Pull request closed without merging**